node-backend/app/controllers/v1/recognitionController.js

```diff
@@ -241,6 +236,19 @@ const getFilterData = (data) => {
   return filterData;
 };
 
+const createWhereClause = ({ orgId }, id) => {
```

Comment:
what does id stands for?

Reply:
optional param like(core_value_id,given_by,given_for) All are id.